config/services/balena-registry.service

```diff
@@ -1,17 +0,0 @@
-[Unit]
-Description=balena-registry
-Requires=confd.service balena-root-ca.service
-After=confd.service balena-root-ca.service
```

Comment:
I'd like you to confirm that the no-systemd base image actually handles installing the root CA cert, please. A sure fire way to do it would be to provision an openBalena instance (that uses this registry tag) and a device.

Reply:
Thanks for the tip. It definitely needed another change in the base image to make sure we run the root-ca script as well...

with that I'm getting:
```
❱ env BALENARC_BALENA_URL=openbalena.local NODE_EXTRA_CA_CERTS=./config/certs/root/ca.crt balena-cli deploy testapp hello-world
[Info]    Creating default composition with image: "hello-world"
[Info]    Everything is up to date (use --build to force a rebuild)
[Info]    Creating release...
[Info]    Pushing images to registry...
[Info]    Saving release...
[Success] Deploy succeeded!
[Success] Release: ad424dd3b38ffaebd9be1e8725ec7034
```